### PR TITLE
Add Html and Css to page.

### PR DESCRIPTION
**Pull request by DiamondNath07**
### Testing Microverse Linters
**This pull request attempts to test microverse linters**

### A quick grasp of the files added includes;

- Add index.html
- Add styles.css
- Add .gitignore
- Created a .github folder/workflows where I added the linters.yml file also
- Add text.md
- Add .stylelintrc.json

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1 +1,64 @@
...
+
+[Live Demo Link](https://livedemo.com)
+
+## Getting Started
+
+**In order to use this project, you have to create a .githhub/workflow folder on your root directory and add this file 'linters.yml'. Then checkout to your feature branch and compare with your main branch and then check linters for error on your feature branch.**
+
+---
+
+To get a local copy up and running follow these simple example steps.
```
Comment (franclobo) [inline readme.md]: - [ ] Please 🙏🏽 replace the information with your instructions to use your repository, for example:
- Clone the repo and open it in your browser.

File: readme.md [hunk 2]
```diff
@@ -1 +1,64 @@
...
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
```
Comment (franclobo) [inline readme.md]: - [ ] Kindly delete the sections that you are not using yet.

File: readme.md [hunk 3]
```diff
@@ -1 +1,64 @@
...
+
+## Authors
+
+👤 **Author1**
```
Comment (franclobo) [inline readme.md]: - [ ] Please 🙏🏽 add your name to your repository,

File: readme.md [hunk 4]
```diff
@@ -1 +1,64 @@
...
+
+- GitHub: [@githubhandle](https://github.com/DiamondNath07)
+- Twitter: [@twitterhandle](https://twitter.com/@diamondNath07)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/diamond-nathaniel-6b664b245)
```
Comment (franclobo) [inline readme.md]: - [ ] Kindly replace the links with your contact information.

File: readme.md [hunk 5]
```diff
@@ -1 +1,64 @@
...
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- @BertrandConxy
+- www.microverse.com
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (franclobo) [inline readme.md:56]: - [ ] Please 🙏🏽 make sure that all the links work as expected, we can use `[MIT](./LICENSE)`. First of all, kindly add the LICENSE file to your repository.

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @DiamondNath07, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- No linter errors ✔️ 
- Descriptive PR ✔️ 
- Descriptive commits 💯 


 ## Required Changes ♻️
_Check the comments under the review._

# - This is a re-review but I can't see the previous code review.
- [ ] For a correct Github flow please 🙏🏽 make sure to do all the changes in the correct branch don't do it in the main branch.
image


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rukundoeric — CHANGES_REQUESTED** [review]
Hello @DiamondNath07 👋,

_This is an additional review as a result of the second opinion request._

_Great job so far 👏 💯 👍. However, there are few issues we need to address before we move on. Please, check the comments below._

## Required Changes ♻️
 - [ ] Kindly refer to the previous required changes and implement them.
 - [ ] Also, it would be nice if you rebased your branch to remove those conflicts.
 - [ ] One more thing is to try not to use anther PR as it might confuse the next Reviewer.

Keep rocking 🚀! // can be customized

**Review by sajid-munawar — APPROVED** [review]
### Hi @DiamondNath07 , 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and description :heavy_check_mark:
 
> ## [Optional] suggestions:
> * [ ]  Please check my inline comments.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**